perl/lib/Seq/Tracks/Cadd.pm

```diff
@@ -45,12 +45,33 @@ sub get {
   # Alternatively the CADD data for this position may be missing (not defined)
   # It's slightly faster to check for truthiness, rather than definition
   # Since we always either store undef or an array, truthiness is sufficient
+
+  # Similar to VCF tracks, we only return exact matches
+  # So we do not tile across the entire deleted / inserted region
+  # We return only the first result
+  # In the case of a SNP there is only 1 result
+  # In the case of an indel, we return undef only once
+  if ( $_[5] > 0 ) {
+    return $_[6];
+  }
+
   if ( !defined $_[1]->[ $_[0]->{_dbName} ] ) {
     $_[6][ $_[5] ] = undef;
 
     return $_[6];
   }
 
+  # For indels, we return undef
```

Comment:
Is 'undef' a standard within CADD or other databases that might be expected by users or would NA/nan make sense in these cases too?

Reply:
Undef is the way that bystro internally handles missing values; I'll make sure it's described better

Reply:
You don't/can't use the missing character, because then we would mix functionality (we have a dedicated Output creator, and we want multiple possible Output backends)